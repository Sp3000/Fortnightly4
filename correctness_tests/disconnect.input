place 0 0 red
place 0 1 red
count
connect 0 0 0 1
count
disconnect 0 1 0 0
count
connect 0 1 0 0
count
disconnect 0 1 0 0
count
connect 0 0 0 1
place 1 0 green
place 1 1 red
count
connect 1 1 1 0
count
connect 1 0 0 0
connect 1 1 0 1
count
disconnect 0 0 1 1
disconnect 1 1 1 2
disconnect 1 0 0 0
count
disconnect 0 0 0 1
count
halt