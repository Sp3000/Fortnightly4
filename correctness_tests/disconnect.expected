2
1
2
1
2
3
2
1
Error: disconnect 0 0 1 1
Error: disconnect 1 1 1 2
1
2
